### PR TITLE
MVP

### PR DESCRIPTION
**Pull request by lisaMTayl**

### CODE REVIEW

File: ANSWERS.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Scope.  Application state is global, component state is local.  In react the store is application state and available to all, whereas component state is only valid and available in that particular component.
+
+
+- [ ] Describe `redux-thunk`, what does it allow us to do? How does it change our `action-creators`?
+
+Redux Thunk is delaying evaluation.  Officially, it is a middleware that lets you call action creators that return a function instead of an action object. That function receives the store’s dispatch method, which is then used to dispatch regular synchronous actions inside the body of the function once the asynchronous operations have completed.
+  
+- [ ] What is your favorite state management system you've learned and this sprint? Please explain why!
+
+Context.  It's the only class I was able to attend live this week.
```
Comment (AdetunjiShennaike) [inline ANSWERS.md:28]: try to redo the sprint with context

File: smurfs/src/store/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      console.log(err);
+      dispatch({ type: POST_SMURF_FAILURE, payload: err.response })
+    })
+};
+
+export const DELETE_SMURF_START = 'DELETE_SMURF_START';
+export const DELETE_SMURF_SUCCESS = 'DELETE_SMURF_SUCCESS';
+export const DELETE_SMURF_FAILURE = 'DELETE_SMURF_FAILURE';
+
+export const deleteSmurf = id => dispatch => {
```
Comment (AdetunjiShennaike) [inline smurfs/src/store/actions/index.js]: this is good

File: smurfs/src/store/reducers/index.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        addingSmurf: false,
+        error: action.payload
+      };
+    case DELETE_SMURF_START:
+      return {
+        ...state,
+        deletingSmurf: true,
+        error: "",
+      };
+    case DELETE_SMURF_SUCCESS:
```
Comment (AdetunjiShennaike) [inline smurfs/src/store/reducers/index.js]: this is good also

File: smurfs/src/components/SmurfList.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import Smurf from './Smurf';
+
+const SmurfList = props => {
+
+  const getSmurfs = e => {
+    e.preventDefault();
+    props.getSmurfsData();
+  };
+
+
```
Comment (AdetunjiShennaike) [inline smurfs/src/components/SmurfList.js:20]: you would probably add the delete button here as well as the logic that calls the action. 
this way you can directly grab the id off the smurf and have it get passed in to the function without having to sift through data. 
also add the smurf list here haha

File: smurfs/src/components/SmurfForm.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
```
Comment (AdetunjiShennaike) [inline smurfs/src/components/SmurfForm.js:1]: ADD COMMENTS!!!

**Review by AdetunjiShennaike — APPROVED** [review]

File: ANSWERS.md (hunk 1 above)
Comment (lisaMTayl) [reply ANSWERS.md:28]: It's on my list

**Comment by lisaMTayl** [comment]
[Smurf Spring](https://trello.com/c/nXuNa8LV/1-smurf-spring)

**Pull request merged**